contracts/utils/cryptography/draft-ERC7739Utils.sol

```diff
@@ -217,36 +214,33 @@ library ERC7739Utils {
     /**
      * @dev Try to validate the contents type is a valid EIP-712 type.
      *
-     * A valid `contentsType` is considered invalid if it's empty or it:
+     * A valid `contentsType` is considered invalid if it's empty or if:
+     * - it starts with a '('.
+     * - it contains any of the following bytes: , )\x00
      *
-     * - Starts with a-z or (
-     * - Contains any of the following bytes: , )\x00
+     * NOTE: This is a looser take on the ERC very strict restrictions. This part appears to be under discussion, and
+     * therefore the restrictions implemented here may change in a future release.
      */
     function tryValidateContentsType(
```

Comment:
The new logic makes more sense IMO but does not match the ERC so we need the ERC to lift the a-z restriction.

Reply:
SO what do we do, re-add the lowercase restriction?

Reply:
I guess I would keep the logic of the ERC until that gets changed. But I think it needs to be changed before this is released.